addons/modules/XEH_preInit.sqf

```diff
@@ -6,4 +6,15 @@ PREP_RECOMPILE_START;
 #include "XEH_PREP.hpp"
 PREP_RECOMPILE_END;
 
+["ModuleCurator_F", "Init", {
```

Comment:
Comment above this line maybe?
`// Fix copy/pasted modules that use BI's module framework`